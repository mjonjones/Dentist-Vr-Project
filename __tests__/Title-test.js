import renderer from 'react-test-renderer';
import React from 'react';
import Title from '../Title';
import ReactDOM from 'react-dom';

it('renders a snapshot of Title', () => {
    const tree = renderer.create(<Title/>).toJSON();
    expect(tree).toMatchSnapshot();
});

it('renders a snapshot of Title with props', () => {
    let text = "Test";
    const tree = renderer.create(<Title txt={text}/>).toJSON();
    expect(tree).toMatchSnapshot();
});