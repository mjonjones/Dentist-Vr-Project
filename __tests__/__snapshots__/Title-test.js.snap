// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of Title 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#3498db",
      "borderRadius": 0.07,
      "height": 0.15,
      "layoutOrigin": Array [
        0.5,
        0.5,
        0.5,
      ],
      "opacity": 1,
      "position": "absolute",
      "transform": Array [
        Object {
          "rotateY": 0,
        },
        Object {
          "rotateX": 20,
        },
        Object {
          "translate": Array [
            0,
            0,
            -1,
          ],
        },
      ],
      "width": 0.4,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    isOnLayer={false}
    style={
      Object {
        "color": "#fff200",
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    
  </Text>
</View>
`;

exports[`renders a snapshot of Title with props 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#3498db",
      "borderRadius": 0.07,
      "height": 0.15,
      "layoutOrigin": Array [
        0.5,
        0.5,
        0.5,
      ],
      "opacity": 1,
      "position": "absolute",
      "transform": Array [
        Object {
          "rotateY": 0,
        },
        Object {
          "rotateX": 20,
        },
        Object {
          "translate": Array [
            0,
            0,
            -1,
          ],
        },
      ],
      "width": 0.4,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    isOnLayer={false}
    style={
      Object {
        "color": "#fff200",
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    Test
  </Text>
</View>
`;
